src/codegen/mod.rs

```diff
@@ -636,9 +678,10 @@ impl CodeGenerator for Type {
 
                 // If this is a known named type, disallow generating anything
                 // for it too.
-                let spelling = self.name().expect("Unnamed alias?");
-                if utils::type_from_named(ctx, spelling).is_some() {
-                    return;
+                if let Some(spelling) = self.name() {
```

Comment:
Hmm, we're not changing which `Alias` we generate, why is this change needed? It doesn't really make sense to have an unnamed alias.

Reply:
sorry, forget to restore it before commit, my fault.